config/manifests/codeflare-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
ClusterServiceVersion file is now in two places, config/manifests/codeflare-operator.clusterserviceversion.yaml and config/manifests/bases/codeflare-operator.clusterserviceversion.yaml